test/src-tests/index-test.js

```diff
@@ -38,6 +38,20 @@ describe('find-parent/index', function() {
         ancestor.appendChild(parent);
       });
 
+      it('should throw if opts is truthy and not an object', function() {
```

Comment:
Why specifically truthy? That seems like an implementation of the `||`. A test like this should probably be `should throw if opts is not undefined and not an object`